tools/VscPreviewExtension/media/js/htmlUpdate.js

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) Microsoft. All rights reserved. Licensed under the MIT license. See LICENSE file in the project root for full license information.
+$(function () {
+  var port = $("meta[name='port']")[0].content;
+  var pageRefreshFunctionName = "refresh";
+  var filePath = $("meta[name='filePath']")[0].content;
+  var lastLocation = 0;
+  var filePathEscape = filePath.replace(/\\/g, "\\\\");
+  var rightClick = false;
+
+  setInterval(function () {
+    // VSCode bug: https://github.com/Microsoft/vscode/issues/23020
+    // TODO: Remove when bug fixed
+    $("link").each(function (index) {
+      var path = $(this).attr("href");
+      if (path.startsWith("https"))
+        $(this).attr("href", path.replace("https", "http"));
+    });
+
+    $.get("http://localhost:" + port.toString() + "/previewContent")
```

Comment:
>"http://localhost:" + port.toString() [](start = 10, length = 37)

multiple places use same const.....construct a method to pass path such as "previewContent" to return the right path